.github/workflows/deploy-gcp-tests.yml

```diff
@@ -150,6 +147,7 @@ jobs:
           --machine-type ${{ env.MACHINE_TYPE }} \
           --scopes cloud-platform \
           --metadata=google-monitoring-enabled=true,google-logging-enabled=true \
+          --metadata-from-file=startup-script=.github/workflows/scripts/gcp-vm-startup-script.sh \
```

Comment:
I tested this script on the Debian bullseye image. Since we're not seeing any of the output from the script, we can't be sure that it worked.

So I'd prefer to use the bullseye image for the instance, and I'll also add a check that it worked below.

Reply:
I SSHed into the VM to confirm this change was applied, but using Debian should not hurt so I'm adding it here: 
[chore: update our VM image to bullseye](https://github.com/ZcashFoundation/zebra/pull/5367/commits/ce80c3af7e36bde0f4cb51ef12106c9cd65cb6be)

Reply:
Thanks for letting me know, can you mention any manual checks you've done in the PR description next time?